### DISCUSSION TITLE
Need help while using One Click DigitalOcean deploy

### DISCUSSION THREAD

**Discussion by ashx404** [directus/directus#2846] [Archive]
I have deployed Directus earlier on Google Cloud Platform and then I had to use Cloud SQL and Cloud Storage apart from the Directus Instance. Now I am trying to setup it on Digital Ocean and I am unable to find the storage settings. How will Files will be saved in DigitalOcean?

**Comment by rijkvanzanten** [comment]
DO Spaces supports the S3 APIs, so you can use the regular S3 storage adapter and connect it to DO spaces

**Reply by ashx404** [reply]
And the SQL is included in the Droplet, right?

**Reply by rijkvanzanten** [reply]
Depends on how you configure your droplet. You can run MySQL on the droplet next to Directus, or use DO's managed database service
